RecoLocalCalo/HGCalRecProducers/python/HGCalRecHit_cfi.py

```diff
@@ -79,8 +79,8 @@
 
     # EM Scale calibrations
     layerWeights = dEdX_weights,
-    thicknessCorrection = cms.vdouble(0.964,0.920,0.909), # 100, 200, 300 um
-    
+    thicknessCorrection = cms.vdouble(1.132,1.092,1.084), # 100, 200, 300 um
```

Comment:
@perrotta We could remove the related commit if desired or could provide plots to support the updated corrections (hopefully by tomorrow).